src/components/Kv/KvCodeBlock.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+	<div class="code" :class="`${nowrap ? 'nowrap' : ''}`">
+		<pre v-if="code">{{ code }}</pre>
+		<slot v-else></slot>
```

Comment:
Please wrap whatever is rendered in a <code> element.

Reply:
Cool, done!